DirectProgramming/Fortran/openmp_samples/sample.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "openmp_samples",
+  "categories": [ "Toolkit/Intel® oneAPI HPC Toolkit" ],
+  "description": "Fortran Tutorial - Using OpenMP",
+  "toolchain": [ "ifort" ],
+  "languages": [ { "fortran": {} } ],
+  "targetDevice": [ "CPU" ],
+  "os": [ "darwin" ],
+  "builder": [ "make" ]
+}
```

Comment:
Please, add the `"ciTests"` field according to https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object

Reply:
@greenrongreen  This part is what is preventing this merge mostly. Please add the ciTests field as mentioned by Roman.

Reply:
added the CI stanzas and pushed the 3 json files